api/views/__init__.py

```diff
@@ -40,9 +44,10 @@
     ProfileSerializer,
     RelayAddressSerializer,
     UserSerializer,
+    WebcompatIssueSerializer,
 )
 
-
+info_logger = logging.getLogger("eventsinfo")
```

Comment:
question: Is this the only `eventsinfo` logger in the code-base? Will that work fine with out existing log and logger?

Reply:
The `eventsinfo` logger has been around for a while, see https://github.com/mozilla/fx-private-relay/pull/1422. It's been used in other parts of the code-base to log non error events, for example in email tracker Foxfooding logs, bounce handling logs, and more.